tests/Unit/Libraries/Cms/Form/Rule/FolderPathExistsRuleTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package     Joomla.UnitTest
+ * @subpackage  Rule
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Tests\Unit\Libraries\Cms\Form\Rule;
+
+use Joomla\CMS\Form\Rule\FolderPathExistsRule;
+use Joomla\Tests\Unit\UnitTestCase;
+
+/**
+ * Test class for FolderPathExistsRule.
+ *
+ * @package     Joomla.UnitTest
+ * @subpackage  Form
+ * @since       __DEPLOY_VERSION__
+ */
+class FolderPathExistsRuleTest extends UnitTestCase
+{
+	/**
+	 * Test data for the testRule method
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function dataTest(): array
+	{
+		$xml = new \SimpleXMLElement('<field
+			name="file_path"
+			type="text"
+			label="COM_MEDIA_FIELD_PATH_FILE_FOLDER_LABEL"
+			description="COM_MEDIA_FIELD_PATH_FILE_FOLDER_DESC"
+			size="50"
+			default="images"
+			validate="folderPathExists"
+			exclude="administrator|media"
+		/>'
+		);
+
+		return [
+			[true, $xml, ''],
```

Comment:
```suggestion
			[true, $xml, ''],
			[false, $xml, JPATH_ROOT],			
```

Add additional test for "Joomla root is not allowed"